src/vuex/modules/intercom.js

```diff
@@ -14,7 +14,7 @@ export default () => {
     },
     actions: {
       displayMessenger({ state }) {
-        state.intercom.displayMessenger()
+        if (config.mobileApp) state.intercom.displayMessenger()
```

Comment:
I don't understand. This seems like it still would only work on mobile

Reply:
And I am right

<img width="480px" src="https://user-images.githubusercontent.com/40721795/74083197-8ca30b00-4a61-11ea-84ae-48f384025e77.png">

Did this work on your laptop @faboweb ?

Reply:
It didn't create an error! "displayMessenger" is only for mobile
For desktop there is a class defined in index.html (line 84)

Reply:
Ok, I just saw it.
Quite strangely both messages in PageValidator and PageTransactions are now gone O.o
What happened? Overridden by other PR I guess?

Reply:
sadly no, Intercom only works on certain domains.